logconv.m

```diff
@@ -61,8 +61,8 @@ function logconv()
 
 % only plot the logged data from t_start to t_end. If both are set to 0.0
 % all the logged data is plotted.
-t_start = 0.0;
-t_end = 0.0;
+t_start = 640.0;
+t_end = 650.0;
```

Comment:
should be changed back to the default value

Reply:
Already changed back